include/glow/Partitioner/Partitioner.h

```diff
@@ -30,6 +30,17 @@ using ComputeTimeMapTy = std::unordered_map<Node *, float>;
 using NodesSetTy = std::set<Node *>;
 using PartitionCostMapTy = llvm::DenseMap<Function *, GraphMemInfo>;
 
+/// Data structure that contains the info for each type of backend used for
+/// partitioning.
+struct BackendInfo {
+  // Num of the devices which has the same type of backend.
```

Comment:
Members should be commented with three slashes '///'. This will allow Doxygen to keep the comment in the docs.